theatre/studio/src/panels/SequenceEditorPanel/layout/tree.ts

```diff
@@ -105,29 +113,35 @@ export const calculateSequenceEditorTree = (
 
     if (Object.keys(trackSetups).length === 0) return
 
+    const isCollapsedP =
+      collapsableP.byId[createStudioSheetItemKey.forSheetObject(sheetObject)]
+        .isCollapsed
+    const isCollapsed = valueDerivation(isCollapsedP).getValue() ?? false
+
     const row: SequenceEditorTree_SheetObject = {
       type: 'sheetObject',
+      isCollapsed: isCollapsed,
       top: topSoFar,
       children: [],
       depth: level,
       n: nSoFar,
       sheetObject: sheetObject,
-      nodeHeight: heightOfAnyTitle,
+      nodeHeight: HEIGHT_OF_ANY_TITLE,
       heightIncludingChildren: -1,
     }
     arrayOfChildren.push(row)
     nSoFar += 1
-    topSoFar += heightOfAnyTitle
-
-    addProps(
-      sheetObject,
-      trackSetups,
-      [],
-      sheetObject.template.config,
-      row.children,
-      level + 1,
-    )
-
+    topSoFar += HEIGHT_OF_ANY_TITLE
```

Comment:
Suggestion: add comments for what each addition is, in this case it's for the height of the sheet object row header, I think, right?